Fixed bad extraction for authorName and authorDescription for bunch of domains

abhishekvm opened the pull request:
----------
http://www.networkcomputing.com/data-centers/why-dbas-need-networking-skills/1883902615

Before
"author": "s"
"author_description": "s"

After
"author": "Thomas LaRock",
"author_description": "Thomas LaRock"

----------
http://www.marketingprofs.com/articles/2015/28657/unlock-mobile-advertising-by-measuring-the-real-world

Before
"author": John Bus is SVP of customer insights at Marchex

After
"author": John Busby is SVP of customer insights at Marchex

----------
https://www.ge.com/digital/blog/devops-no-royal-use-we

Before
"author": "Senior Technical Writer at GE",

After
"author": Denny Bulcao, Jr.

----------
http://www.enterpriseinnovation.net/article/big-data-survive-intensifying-squeeze-regulatory-compliance-1955902423

Before
"author": "",
"author_description": "",

After
"author": "Jason Demby"
"author_description": "By Jason Demby, \u200Edirector of Business Development of Financial Services, Datameer | 2016-05-10"

Commit:
Fixed bad extraction for authorName and authorDescription for bunch of domains

----------
http://www.networkcomputing.com/data-centers/why-dbas-need-networking-skills/1883902615

Before
"author": "s"
"author_description": "s"

After
"author": "Thomas LaRock",
"author_description": "Thomas LaRock"

----------
http://www.marketingprofs.com/articles/2015/28657/unlock-mobile-advertising-by-measuring-the-real-world

Before
"author": John Bus is SVP of customer insights at Marchex

After
"author": John Busby is SVP of customer insights at Marchex

----------
https://www.ge.com/digital/blog/devops-no-royal-use-we

Before
"author": "Senior Technical Writer at GE",

After
"author": Denny Bulcao, Jr.

----------
http://www.enterpriseinnovation.net/article/big-data-survive-intensifying-squeeze-regulatory-compliance-1955902423

Before
"author": "",
"author_description": "",

After
"author": "Jason Demby"
"author_description": "By Jason Demby, \u200Edirector of Business Development of Financial Services, Datameer | 2016-05-10"

skyshard commented on src/test/resources/de/jetwick/snacktory/cracked.html:
```diff
@@ -1,212 +1,212 @@
-
-												
-
+mar<!DOCTYPE html>
```
is this line intentional

abhishekvm replied:
This was unintentional. I have reverted it.

Commit:
Fixed authorName, authorDescription, publicationDate for it-business

-----------
http://www.it-business.de/cloud-stellt-kleine-mit-grossen-haendlern-gleich-a-551169/

Before
"author": "Vogel Business Media GmbH & Co. KG",
"author_description": "Vogel Business Media GmbH & Co. KG",
"publishDate": null,

After
"author": "Enrique Nuñez"
"author_description": "Enrique Nuñez ist seit mehr als 20 Jahren im Internet tätig und hat in dieser Zeit drei Startup-Unternehmen gegründet."
"publishDate": "2016-09-22 00:00:00"

Commit:
Fixed authorName, authorDescription for bulldogreporter.com

-----------
https://www.bulldogreporter.com/are-you-guilty-of-pr-data-bias-what-why-and-how-to-check/

Before
"author": "Related",
"author_description": "Related",

After
"author": "Kelly Byrd"
"author_description": "By Kelly Byrd, PR Engineer,""

Commit:
Reverting unintentional changes

andresp99999 reviewed (approved):
LGTM. One minor comment, it may be good to start splitting the unit tests in separate files (the ArticleTextExtractor.java is is pretty large).
